tools/arxiv/tests/run_large_scale_test.sh

```
@@ -46,32 +46,31 @@ echo "Working directory: $PROJECT_ROOT"
-cd "$ARXIV_TOOLS/scripts"
+cd "$ARXIV_TOOLS/utils"
 
-if [ ! -f "data/arxiv_collections/arxiv_ids_*.txt" ]; then
-    echo "Running paper collection script..."
-    python collect_ai_papers.py
-    
-    # Check if we got enough papers
-    PAPER_COUNT=$(wc -l < data/arxiv_collections/arxiv_ids_*.txt | head -1)
-    echo -e "${GREEN}Collected $PAPER_COUNT unique papers${NC}"
+# Use compgen to safely check for matching files
+PAPER_LIST_FILES=($(compgen -G "../../../data/arxiv_collections/arxiv_ids_*.txt"))
```

Shellcheck (0.10.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)